### ISSUE TITLE
Compilation Fails v0.3.0

### ISSUE THREAD

**Issue by ozburo** [Milad-Akarie/auto_route_library#36]
Just updated to new v0.3.0 and can't build Flutter app anymore.

```
Compiler message:
../../../flutter/.pub-cache/hosted/pub.dartlang.org/auto_route-0.3.0/lib/src/extended_navigator.dart:79:56: Error: This expression has type 'void' and can't be used.
  bool pop<T extends Object>([T result]) => _navigator.pop<T>(result);
```

My Flutter version:

```
[✓] Flutter (Channel dev, v1.15.3, on Mac OS X 10.14.6 18G103, locale en-US)
 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.3)
[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
[✓] Android Studio (version 3.5)
[✓] VS Code (version 1.42.1)
[✓] Connected device (1 available)
```

**Comment by htueko** [comment]
**same error here**
```
Error compiling dartdevc module:auto_route|lib/auto_route.ddc.js

packages/auto_route/src/extended_navigator.dart:79:56: Error: This expression has type 'void' and can't be used.
  bool pop<T extends Object>([T result]) => _navigator.pop<T>(result);
                                                       ^
AssetNotFoundException: auto_route|lib/auto_route.ddc.js
Failed after 35.2s
Finished with error: Failed to build application for the Web.
```

```
auto_route: ^0.3.0
auto_route_generator: ^0.3.0
build_runner: ^1.7.4
```

```
[✓] Flutter (Channel dev, v1.15.3, on Linux, locale en_GB.UTF-8)
    • Flutter version 1.15.3
    • Framework revision 67826bdce5 (3 weeks ago), 2020-02-10 14:59:32 -0800
    • Engine revision 6158f03ef5
    • Dart version 2.8.0 (build 2.8.0-dev.8.0 514a8d4c84)
```

**Comment by karanianandkumar** [comment]
@htueko ,Did you find any solution?

**Comment by htueko** [comment]
@karanianandkumar Not yet

**Comment by Milad-Akarie** [comment]
Fixed in v0.3.1
Thank you all for your efforts.

**Comment by ozburo** [comment]
Thank you for taking care of this so quickly 👍
